web/src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -122,6 +122,7 @@
 					+ "<br />phoneNumberTooLong                            : Phone number length should not exceed 15 characters."
 					+ "<br />invalidPhoneNumber                            : Phone number should not contain alphabetic characters."
 					+ "<br />participateModuleNotActive                    : Participant Module is Not Active."
+					+ "<br />participantIDAlreadyExists					   : Participant id is already exists."
```

Comment:
remove the extra "is" and capitalize ID so it reads - "Participant ID already exists."

Reply:
Changed as per the comments